data/users.js

```diff
@@ -0,0 +1,94 @@
+const mongoCollections = require('../config/mongoCollections');
+const users = mongoCollections.users;
+const uuid = require('uuid');
+
+let exportedMethods = {
+
+  //search for a user by ID
+  async getUserById(id) {
+    const userCollection = await users();
+    const user = await userCollection.findOne({ _id: id });
+    if (!user) throw 'User not found';
+    return user;
```

Comment:
A quick-tip for testing: You can keep this code for now(commented). But for testing purposes, you can return dummy data. Later on, update this.